libmate-desktop/mate-colorsel.c

```diff
@@ -2756,8 +2740,8 @@ mate_color_selection_palette_from_string (const gchar *str,
               done = FALSE;
             }
 
-          retval = g_renew (GdkColor, retval, count + 1);
-          if (!gdk_color_parse (start, retval + count))
+          retval = g_renew (GdkRGBA, retval, count + 1);
+          if (!gdk_rgba_parse (retval + count, start))
```

Comment:
Do you need to handle the case when there's no `#` prefix like done above?

Reply:
```#``` prefix needs to be preserved

Reply:
You mean the value in parses *has* to be formatted in a way compatible with `gdk_rgba_parse()`?

Reply:
I guess it's fine, so let's leave this alone.